chainer/backends/cuda.py

```diff
@@ -281,12 +281,16 @@ def _array_to_gpu(array, device, stream):
     assert device is DummyDevice or isinstance(device, Device)
     if array is None:
         return None
+
     if isinstance(array, (numpy.number, numpy.bool_)):
         array = numpy.asarray(array)
+    elif isinstance(array, intel64.mdarray):
+        # ideep.mdarray to numpy.ndarray
```

Comment:
I feel this comment does not explain anything but what is written in the code.

Reply:
Fixed.